evc-sumo/docker/get-system-version.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#  Copyright (C) 2023 LEIDOS.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+SYSTEM_TAG_PREFIX="CARMASystem"
```

Comment:
Should be `SYSTEM_TAG_PREFIX="carma-system"`, this might be based off an out-of-date copy of these cross-repo files. I'd recommend checking these against the copies in carma-platform, those should be the latest. If these are what's in the root of this repo, we should make an issue to update those prior to the next release.